packages/common/src/components/AppHeader/AppHeaderImageWithText.tsx

```diff
@@ -4,18 +4,19 @@ import styled from 'styled-components'
 interface Props {
   items?: React.ReactNode
   image?: string
+  alt?: string
 }
 
 const Wrapper = styled.div`
   border-right: solid 1px #01a5a3;
   padding-right: 15px;
 `
 
-const AppHeaderImageWithText: React.FC<Props> = ({ image, items }) => {
+const AppHeaderImageWithText: React.FC<Props> = ({ image, items, alt }) => {
   return (
     <>
       <div role="presentation" className="iu-mr-300 iu-ml-400">
-        <img src={image} className={'iu-mr-200'} />
+        <img src={image} className={'iu-mr-200'} alt={alt} />
       </div>
       <Wrapper className="iu-lh-narrow">
         <div className="">{items}</div>
```

Comment:
Not related to this PR.

Reply:
We should take the opportunity to improve the code while working with it, even if it not directly related to the current change. It's a common development principle to keep the code clean and maintainable and stave off technical debt.

Reply:
I agree if it is a change that is risk free and clearly necessary. In this case I don't think it's obvious if it should be removed or not since it is taken from the UX-framework.

Reply:
I think the documentation is pretty straight forward

> Writing `<h2 role="presentation">Democracy Dies in Darkness</h2>` removes the heading semantics of the `<h2>` element, making it the equivalent of `<div>Democracy Dies in Darkness</div>`. The heading role semantics are removed, but the content itself is still available.

and

> For elements with no required children, any elements nested inside the element with `role="presentation"` or `role="none"` preserve their semantics.

https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/presentation_role

Reply:
Nevermind my previous comment, I misinterpreted your reply!